docs/ops.html

```diff
@@ -3992,6 +3979,75 @@ <h3>Finalizing the migration</h3>
 
 # Other configs ...</pre>
 
+  <h3>Reverting to ZooKeeper mode During the Migration</h3>
+  <p>
+    While the cluster is still in migration mode, it is possible to revert to ZK mode.  The process
+    to follow depends on how far the migration has progressed. In order to find out how to revert,
+    select the <b>final</b> migration step that you have <b>completed</b> in this table.
+  </p>
+  <p>
+    Note that the directions given here assume that each step was fully completed, and they were
+    done in order. So, for example, we assume that if "Enabling the migration on the brokers" was completed,
+    "Provisioning the KRaft controller quorum" was also fully completed previously.
+  </p>
+  <p>
+    If you did not fully complete any step, back out whatever you have done and then follow revert
+    directions for the last fully completed step.
+  </p>
+
+  <table class="data-table">
+      <tbody>
+      <tr>
+        <th>Final Migration Section Completed</th>
+        <th>Directions for Reverting</th>
+        <th>Notes</th>
+      </tr>
+      <tr>
+        <td>Preparing for migration</td>
+        <td>The prepartion section does not involve leaving ZK mode. So there is nothing to do in
+            the case of a revert.</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Provisioning the KRaft controller quorum</td>
+        <td>Deprovision the KRaft controller quorum, and then you are done.</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Enabling zookeeper.metadata.migration.enable on the brokers</td>
+        <td>Roll the broker cluster with zookeeper.metadata.migration.enable=false. Then,
+            deprovision the KRaft controller quorum. Then you are done.</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Migrating brokers to KRaft</td>
+        <td><p>
+            Roll the broker cluster with the process.roles configuration omitted, node.id
+            replaced with broker.id, and the zookeeper.connect configuration set to a valid
+            value.
+          </p>
+           <p>
+            After this roll is fully complete, perform a second roll where you set
+            zookeeper.metadata.migration.enable=false. Then,
+            deprovision the KRaft controller quorum. Then you are done.
+          </p>
+        </td>
+        <td>
+            Make sure that on the first roll, zookeeper.metadata.migration.enable remains set to
+            true. <b>Do not set it to false until the second roll.</b>
```

Comment:
I think we can remove this sentence if we take the comment I made above (the one about having zookeeper.metadata.migration.enable=true otherwise you get an exception on broker restart).
Also, imagine a user following this steps one after the other, when they reach this point, it's too late to get the note.

Reply:
Well, the notes are intended to be read before you perform the revert :)

Take a look at the latest version and see what you think.